ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,61 +13,75 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class="has-top-margin-xl">
-  <p class="is-label">Which secrets would you like us to sync?</p>
-  <p class="sub-text">
-    Select a KV engine mount and path to sync a secret to the destination.
-  </p>
-
+<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
   <MessageError @errorMessage={{this.error}} />
 
+  {{#if this.syncedSecret}}
+    <Hds::Alert @type="inline" @color="success" as |A|>
```

Comment:
Should this be dismissable, or maybe time out since they can continue syncing more secrets from the page?

Reply:
I reached out to design to get clarification and will update if need be.

Reply:
The feedback was that manual dismiss might take away from the workflow and a timeout based solution may not be ideal either so I've updated to hide the banner on each submit action and then it will be rendered on success with the synced secret path in the message.